examples/auth0/components/layout.js

```diff
@@ -7,27 +7,27 @@ function Layout({ user, loading = false, children }) {
       <Head>
         <title>Next.js with Auth0</title>
       </Head>
-
-      <Header user={user} loading={loading} />
-
+      <Header user={user} />
```

Comment:
Why was `loading` removed from `Header` ?

Reply:
Since loading is validated in the Layout component, it doesn't make much sense to have it in the header anymore. Anyway, I can put it back if you want

Reply:
That's intended, not all pages have to handle the loading state in the same way.